apps/exts/cesium.performance.app/cesium/performance/app/extension.py

```diff
@@ -1,12 +1,33 @@
 from functools import partial
 import asyncio
-from typing import Optional
+import time
+from typing import Callable, List, Optional
 import logging
+import carb.events
 import omni.ext
 import omni.ui as ui
+import omni.usd
+import omni.kit.app as app
 import omni.kit.ui
+from omni.kit.viewport.utility import get_active_viewport
+from pxr import Gf, Sdf, UsdGeom
 from .performance_window import CesiumPerformanceWindow
+from cesium.omniverse.bindings import acquire_cesium_omniverse_interface, release_cesium_omniverse_interface
 from cesium.omniverse.utils import wait_n_frames, dock_window_async
+from cesium.usd.plugins.CesiumUsdSchemas import (
+    Data as CesiumData,
+    Georeference as CesiumGeoreference,
+    Imagery as CesiumImagery,
+    Tileset as CesiumTileset,
+    Tokens as CesiumTokens,
+)
+
+ION_ACCESS_TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJqdGkiOiIyZTA0MDlmYi01Y2RhLTQ0MjQtYjBlOS1kMmZhMzQ0OWRkNGYiLCJpZCI6MjU5LCJpYXQiOjE2ODU2MzExMTF9.y2CrqatkaHKHcj6NIDJ8ioll-tnOi-2CblnzI6iUays"  # noqa: E501
+GOOGLE_3D_TILES_URL = "https://tile.googleapis.com/v1/3dtiles/root.json?key=AIzaSyC2PMYr_ZaMJT5DdZ8WJNYMwB0lDyvx5q8"
```

Comment:
Thought for later. We ought to set up a way to use environment variables for this sort of thing, rather than hard coding it in the code. Easier to rotate values when we need to do so.

Reply:
Yeah, that would be better. I don't like hardcoding it either. I'll have a follow up PR for that.